concourse/tasks/update-task-definition.yml

```diff
@@ -10,7 +10,11 @@ inputs:
   - name: govuk-infrastructure
     path: src
   - name: release
+    optional: true
   - name: app-terraform-outputs
+  - name: app-image
+    path: image
```

Comment:
Why customise the path? Wouldn't it be simpler to let it default to placing the file in `app-image`, and then using `[ -f "app-image/digest" ]` in the script? Or using `input_mapping: { image: static-image }` if you prefer that?

Reply:
I can do that